lib/errors.ts

```diff
@@ -149,6 +149,8 @@ const EXPECTED_ERROR_REGEXES = [
 	/^BalenaDeviceNotFound/, // balena-sdk
 	/^BalenaExpiredToken/, // balena-sdk
 	/^BalenaInvalidDeviceType/, // balena-sdk
+	/Cannot deactivate devices/i, // balena-api
+	/Devices must be offline in order to be deactivated\.$/i, // balena-api
```

Comment:
These still don't cover the case of answering 'No' to the warning prompt, right?

Reply:
@pdcastro yeah it shows the `Additional information...` message. Let me see if I can find it quickly

Reply:
I guess (didn't test) the fix is to replace `Error` with `ExpectedError` on this line:
https://github.com/balena-io/balena-cli/blob/v12.30.1/lib/utils/patterns.ts#L162

Reply:
@pdcastro it comes from https://github.com/balena-io/balena-cli/blob/529b98552cf7abbecc9dbb58fa32734805767251/lib/utils/patterns.ts#L142-L168 and I tried to do `await patterns.confirm(options.yes, deactivationWarning, '', true);` but same behaviour, prints `Additional information may...` I'll dig into this tomorrow. My guess is that this also happens elsewhere.